wikipedia-sentences-query-while-indexing/flows/dbms.yml

```diff
@@ -8,11 +8,6 @@ pods:
     uses: 'jinahub://TransformerTorchEncoder'
   - name: indexer
     uses: 'jinahub://PostgreSQLStorage'
-    with:
-      hostname: $DB_HOST
-      port: $DB_PORT
-      username: $DB_USERNAME
-      password: $DB_PASSWORD
-      database: $DB_DATABASE
-      table: $DB_TABLE
+    override_with:
+      hostname: host.docker.internal
```

Comment:
this does not make much sense. Why is this `host.docker.internal`?

Reply:
I believe because the flow and the executors it uses are in the same internal docker runtime

Reply:
I think this should not be indented as `hostname` is a flow instead of pod level argument